pulpcore/app/models/task.py

```diff
@@ -247,6 +208,24 @@ def save_heartbeat(self):
         self.save(update_fields=["last_heartbeat"])
 
 
+def _hash_to_u64(value):
+    _digest = blake2s(value.encode(), digest_size=8).digest()
+    return int.from_bytes(_digest, byteorder="big", signed=True)
+
+
+class TaskManager(models.Manager):
+    def filter(self, *args, **kwargs):
+        value = kwargs.pop("reserved_resources_record__resource", None)
+        if value is not None:
+            deprecation_logger.warn(
```

Comment:
```suggestion
            deprecation_logger.warning(
```

nitpick: because `warn` is deprecated in Python.